modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.jsx

```diff
@@ -139,7 +124,7 @@ const RequiredInformation = ({
 			expirationDate:
 				getLicenseKeyEndDatesByLicenseType(infoSelectedKey) ??
 				infoSelectedKey?.selectedSubscription.endDate,
-			licenseEntryType: getLicenseEntryTypeSelected(infoSelectedKey),
+			licenseEntryType: getLicenseKeyEntryTypeSelected(infoSelectedKey),
```

Comment:
can you keep this naming the same as before,`getLicenseEntryTypeSelected`. the objects name is `LicenseEntry` so the rename to `LicenseKeyEntry` doesn't make much sense.